drivers/interrupt_controller/loapic_intr.c

```diff
@@ -205,23 +206,44 @@ static int _loapic_init(struct device *unused)
 	s32_t loApicMaxLvt; /* local APIC Max LVT */
 
 	/* enable the Local APIC */
+#ifndef CONFIG_JAILHOUSE_X2APIC
```

Comment:
lots of ifdefs...could we perhaps make an inline function which does the right thing based on configuration?

/* OR in a bit to the existing value*/
LOAPIC_WRITE(LOAPIC_SVR, LOAPIC_READ(LOAPIC_SVR) | LOAPIC_ENABLE);

Reply:
Ack.

Reply:
loapic_timer.c will be left as is, though, because I see no advantages there.